src/components/ComingSoonSection/ComingSoonMovies.vue

```diff
@@ -0,0 +1,80 @@
+<script>
+import { defineComponent } from "vue";
+import MovieCardBasic from "@/components/Global/MovieCardBasic.vue";
+
+export default defineComponent({
+  name: "ComingSoonMovies",
```

Comment:
Actually, you might omit the `name` here.
If you don't pass it - Vue will use the file name of the SFC as a default (so `ComingSoonMovies`).
That's why naming our files properly is important 😄